openmetadata-service/src/main/java/org/openmetadata/service/airflow/AirflowRESTClient.java

```diff
@@ -248,12 +257,19 @@ public Map<String, String> requestGetHostIp() {
   }
 
   @Override
-  public Map<String, String> getLastIngestionLogs(IngestionPipeline ingestionPipeline) {
+  public Map<String, String> getLastIngestionLogs(IngestionPipeline ingestionPipeline, String after) {
     HttpResponse<String> response;
+    String taskId = TYPE_TO_TASK.get(ingestionPipeline.getPipelineType().toString());
```

Comment:
The task ID is a static mapping. We can do that on the Airflow REST Client directly

Reply:
I think those `task_ids` must be part of our JSON schema definition somewhere. It is hard to know this piece of code is linked to [this](https://github.com/open-metadata/OpenMetadata/blob/b69581299ff74b2b3e9eafdeabdcbfd8e7b8ddb2/openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/profiler.py#L70) one.

Reply:
Thanks, agree. It took me a bit to pick up all this info. Let me open something to map this info. Thanks

Reply:
https://github.com/open-metadata/OpenMetadata/issues/8155